package.json

```diff
@@ -88,8 +87,8 @@
     "supertest": "4.0.2"
   },
   "scripts": {
-    "build": "./node_modules/.bin/babel src -d dist && echo '\n\\033[0;34m[+] \\033[0;32mBuild done\\033[0m'",
-    "build:watch": "onchange 'src/**/*.js' -i -- yarn build",
+    "build": "babel src -d dist",
```

Comment:
I have 2 questions:
- Why do you want to get rid of the `echo`?
- I have never dug into this topic. I don't know why some projects use relative path to use dependencies and why some don't. I actually think it is inconsistent in our own codebase and I know some developers favored the relative path `./node_modules/...` maybe to prevent dependencies conflicts ?

Reply:
About the echo, because babel already outputs a message to tell the build is done, so two of them are displayed
```console
[+] Build done
✨  Done in 3.69s.
```

About removing `node_modules/.bin`, npm/yarn is supposed to put that folder in the PATH anyways, so I don't think it matters.

I had no reason for removing it, I just wanted both build commands to be consistent (which I did a very bad job at: eslint still has the prefix, which jest does not).

Reply:
Also the echo command prevents the users from adding parameters to babel, which is convenient.

For instance: if you want coloring on the output of babel
```
yarn build --highlightCode
```